src/ImportYesod.hs

```diff
@@ -0,0 +1,20 @@
+-- | This module is a convenience module, it re-exports the most common modules
+-- required to write a handler:
+--  - 'DB' exports the @HasDB@ class, which allows you to use 'runDB'
+--  - 'Foundation' exports the actual @Handler@ type as well as the @Foundation@ type
+--  - 'Yesod.Core' provides the majority of the yesod functionality you might require
+--  i.e. the @Html@ ContentType and 'redirect'
+--
+--  A word of caution: It behooves us to keep the re-exports here to a minimum. Choke points
+--  like this in the module graph can really explode compilation times, and the more modules
+--  that get added here, the harder it becomes to manage module cycles.
```

Comment:
This comment should explain most of it, but TL;DR, it becomes really cumbersome to add these 3 exports to every `Handler`, but we also don't want to make this a choke point in the module graph, so best to keep this file minimal.